Dockerfile

```diff
@@ -105,4 +96,22 @@ RUN echo "$ssh_prv_key" > /root/.ssh/id_ed25519 && \
     chmod 600 /root/.ssh/id_ed25519.pub
 
 RUN cd $CODE_DIR &&\
-    git clone git@github.com:event-driven-robotics/EDOPT.git
+    git clone git@github.com:event-driven-robotics/EDOPT.git &&\
+    mv EDOPT object-track-6dof
```

Comment:
could we instead change other references of object-track-6dof to "EDOPT"

Reply:
I updated some paths from `object-track-6dof` to `EDOPT`, in some files, Dockerfile, .ini file, and main.cpp